example_config/MIMIC-50/caml_tune.yml

```diff
@@ -0,0 +1,52 @@
+# data
+data_dir: data/MIMIC-50
+data_name: MIMIC-50
+min_vocab_freq: 3
+max_seq_length: 2500
+
+# train
+seed: 1337
+epochs: 200
+batch_size: 16
+optimizer: adam
+learning_rate: ['choice', [0.001, 0.003, 0.0001, 0.0003]]
```

Comment:
So "choice" means this parameter will be tuned? Is this a keyword designed by us or required by ray? Also this should be mentioned in README